style/ember/README.md

```diff
@@ -2,6 +2,7 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+* Don't use {{partial}} helper. Use a component with only a template.
```

Comment:
I wouldn't mention views at all.

```
Prefer components over partials.
```

Since they're deprecated Ember should warn about those, we shouldn't feel the need to specify it.